src/config.cc

```diff
@@ -2276,7 +2276,7 @@ QString getProgramDataDir() noexcept
 {
   if ( isPortableVersion() )
     return QCoreApplication::applicationDirPath();
-
+// TODO: rewrite this in QStandardPaths::AppDataLocation
   #ifdef PROGRAM_DATA_DIR
```

Comment:
Is this intended?

Reply:
The QStandardPaths::AppDataLocation depends on app name, and we set it to `goldendict-ng`

Changing it needs changes to build scripts :sweat_smile:

https://github.com/xiaoyifang/goldendict-ng/blob/4e1ea82ab722c8c3ba2cbd0a9f5f11771a552a92/src/main.cc#LL272C1-L272C61